nerfstudio/process_data/colmap_utils.py

```diff
@@ -153,12 +154,20 @@ def run_colmap(
     # Bundle adjustment
     sparse_dir = colmap_dir / "sparse"
     sparse_dir.mkdir(parents=True, exist_ok=True)
-    mapper_cmd = [
-        f"{colmap_cmd} mapper",
-        f"--database_path {colmap_dir / 'database.db'}",
-        f"--image_path {image_dir}",
-        f"--output_path {sparse_dir}",
-    ]
+    if glomap_toggle:
+        mapper_cmd = [
```

Comment:
Seems like this could be more consice if you did something like
```python
 mapper_cmd = [
    f"{'glomap' if glomap_toggle else colmap_cmd} mapper",
    f"--database_path {colmap_dir / 'database.db'}",
    f"--image_path {image_dir}",
    f"--output_path {sparse_dir}",
]
```